packages/cxl-ui/scss/global/cxl-marketing-nav.scss

```diff
@@ -32,6 +32,11 @@ vaadin-context-menu-item[theme~="cxl-marketing-nav"] {
     &[focus-ring] {
       box-shadow: none;
     }
+
+    // @see https://app.clickup.com/t/3v0u4ua
+    &::part(checkmark) {
+      width: 1px;
```

Comment:
100%,
but I don't know why its there, and maybe some logic is being performed on it (not sure what), so didn't set `display:none` intentionally.

@anoblet Is testing & reviewing it, maybe its safe to use `display:none`, will see.

Reply:
> but I don't know why its there

"there" means where exactly?

Reply:
`vaadin-context-menu-item` has a part called `checkmark` which inherently takes up space which in the case of the mobile context menu isn't necessary.

Reply:
> > but I don't know why its there
> 
> "there" means where exactly?

I mean, what's the purpose of it, what is the uses case for it, in our `cxl-marketing-nav`, 
in checkboxes, select fields, it might server some purpose.